feat(curriculum): add workshop python functions (#68279)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-kitchen-inventory-tracker/6a3ea771b46a239c67628a07.md

---
id: 6a3ea771b46a239c67628a07
title: Step 15
challengeType: 20
dashedName: step-15
---

# --description--

> Your `make_fried_egg` function has successfully used an egg and checked if the cooking was successful. The last thing it needs to do is give back the updated egg count so the rest of the program knows about it.

Reviewer: ```suggestion
Your `make_fried_egg` function has successfully used an egg and checked if the cooking was successful. The last thing it needs to do is `return` the updated egg count so the rest of the program knows about it.
```

> At the very end of your `make_fried_egg` function (make sure it is not indented inside the `if` or `else` blocks), use the `return` keyword to return your `available_eggs` variable.

Reviewer: ```suggestion
At the bottom of your `make_fried_egg` function, use the `return` keyword to return your `available_eggs` variable. Make sure this line is not indented inside the `if` or `else` blocks.
```
